test/e2e/tests/onboarding/onboarding.spec.ts

```diff
@@ -225,11 +244,12 @@ describe('MetaMask onboarding', function () {
             },
           },
         ],
+        testSpecificMock: tokensMock,
         title: this.test?.fullTitle(),
       },
       async ({ driver, localNodes }) => {
         await localNodes[1].setAccountBalance(
-          '0x0Cc5261AB8cE458dc977078A3623E2BaDD27afD3',
+          IMPORTED_SRP_ACCOUNT_1,
           convertToHexValue(10000000000000000000),
         );
         await importSRPOnboardingFlow({
```

Comment:
given that in this test we import an SRP, we need a custom mock with the account derived from this new srp, because the global mock won't be applied, as it uses the default fixtures account